src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -227,8 +227,13 @@ export default class Pilot {
         this._mcp.setHeadingHold();
 
         const readback = {};
-        readback.log = 'fly present heading';
-        readback.say = 'fly present heading';
+        if (aircraftModel.flightPhase === FLIGHT_PHASE.WAITING) {
```

Comment:
you could get rid of the `if`/`else` and simplify this down to:

```js
const runwayOrPresent = aircraftModel.flightPhase === FLIGHT_PHASE.WAITING ? 'runway' : 'present';
const readback = `fly ${runwayOrPresent} heading`

return [true, readback];
```

Reply:
done